Automattic/jetpack
# Launchpad: Add "Earn money" task to the Newsletter task list

## Proposed changes:


* Adds the "Earn money with your newsletter" to the Free Newsletter task list. Somehow we missed adding it to the Free Newsletter list.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this patch on your Sandbox
* Make sure the public API is sandboxed
* Create a new Free Newsletter or use an existing one.
* Navigate to `/home/:siteSlug`.
* You should see the "Earn money with your newsletter" task
* Make sure you can complete it
